src/Umbraco.Web.UI.Client/package-lock.json

```diff
@@ -1842,8 +1842,7 @@
       "version": "1.5.1",
       "resolved": "https://registry.npmjs.org/base64-js/-/base64-js-1.5.1.tgz",
       "integrity": "sha512-AKpaYlHn8t4SVbOHCy+b5+KKgvR4vrsD8vbvrbiQJps7fKDTkjkDry6ji0rUJjC0kzbNePLwzxq8iypo41qeWA==",
```

Comment:
Don't think you intended this file to be checked in?

Reply:
No, but Im not really sure why this file is in Git in the first place..

Reply:
Then I guess we should just remove from git :)

Reply:
Haha yes both good pages! ;-) But yes, just do a git delete, all good. 👍